### PR TITLE
add user model

### PR DESCRIPTION
**Pull request by ninjacazaam**
Added user model with title bio:text rating:integer price:integer user:references

### CODE REVIEW

**Comment by Jasperinabox** [comment]
Well done : )

**Comment by scallywag89** [comment]
nice, trext

File: db/migrate/20220618060547_create_lives.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class CreateLives < ActiveRecord::Migration[6.1]
+  def change
+    create_table :lives do |t|
+      t.string :title
+      t.text :bio
+      t.integer :rating
+      t.integer :price
+      t.references :user, null: false, foreign_key: true
+
+      t.timestamps
```
Comment (ninjacazaam) [inline db/migrate/20220618060547_create_lives.rb:10]: nice

**Review by ninjacazaam — COMMENTED** [review]
lgtm

**Pull request merged**